weechat_otr_test/mock_weechat.py

```diff
@@ -113,14 +115,14 @@ def restore(self):
 
     def snapshot_weechat_dir(self):
         tar_io = io.BytesIO()
-        with tarfile.open(fileobj=tar_io, mode='w') as tar:
+        with closing(tarfile.open(fileobj=tar_io, mode='w')) as tar:
```

Comment:
You're right. I've changed that and put all the "minor"/style changes into another PR, as you suggested: https://github.com/mmb/weechat-otr/pull/134